Dockerfile

```diff
@@ -12,16 +12,16 @@ COPY go.sum go.sum
 RUN go mod download
 
```

Comment:
same here? https://github.com/kcp-dev/kcp-glbc/blob/main/Dockerfile

Reply:
I'm not seeing the value of that